README.md

```diff
@@ -10,6 +10,19 @@ Please take a look at our issue ticket board to see what we're currently working
 ## Organization and Workflow
 We use a mixed HOOTL, HITL, and Flight-like testing/iteration strategy similar to SpaceX.
 
+## Building and running docker
+1. Download docker desktop: https://www.docker.com/products/docker-desktop
+2. Build docker image
+  - Run: "docker build --tag self-driving-car ."
+3. Running the docker image
+  - When QT works runDocker.sh script will setup enviorment/ permissions and run docker image
+    - Run: "sh runDocker.sh"
+  - For now comment out any display_frame refrences in main control loop
+    - Then run: "docker run self-driving-car config_file_name test_case_name"
+    - For example: "docker run self-driving-car hootl.yaml NoSIM"
```

Comment:
this fails for me...
![image](https://user-images.githubusercontent.com/14984742/157346354-78ccede1-58f2-4cac-8f9e-a306783c59f4.png)

Reply:
did you mean
sudo docker run self-driving-car python3 -m src hootl.yaml NoSIM